.travis.yml

```diff
@@ -24,6 +24,8 @@ before_script:
 cache: pip
 
 deploy:
```

Comment:
You should put this into a separate step and then invoke `pep517.build` in order to build this thing manually.

Reply:
Can you check last commit? I added pep517.build in before_deploy step. It works for me.
What do you think about adding uploading to test.pypi for every commit?

Reply:
You're running the deploy step in each test job, but instead, you should create a separate deploy job on a separate deploy stage.

Reply:
Done, check plz.